File: slackoFrontend/slack/src/app/models/Round.ts
import {Book} from './Book';

export class Round {
  beginningDate: Date;
  consideredBooks: Book[];
  finishDate:	Date;
  id:	number;
  isInitialized: boolean;

  constructor(beginningDate: Date, consideredBooks: Book[], finishDate: Date, id: number, isInitialized: boolean) {
    this.beginningDate = beginningDate;
    this.consideredBooks = consideredBooks;
    this.finishDate = finishDate;
    this.id = id;
    this.isInitialized = isInitialized;
  }
}

File: slackoBackend/src/main/java/com/slack/entities/Round.java
package com.slack.entities;

import lombok.*;
import org.springframework.data.jpa.domain.support.AuditingEntityListener;
import javax.persistence.*;
import java.io.Serializable;
import java.util.Date;
import java.util.List;

@Entity
@Data
@NoArgsConstructor
@EntityListeners(AuditingEntityListener.class)
public class Round implements Serializable {

    @Id
    @Setter(AccessLevel.NONE)
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private Date beginningDate;

    private Date finishDate;

    @ManyToMany(cascade = {CascadeType.MERGE, CascadeType.PERSIST},
                fetch = FetchType.LAZY)
    @JoinTable(name = "rounds_books",
            joinColumns = @JoinColumn(name = "round_id"),
            inverseJoinColumns = @JoinColumn(name = "book_id"))
    private List<Book> consideredBooks;

    private Boolean isInitialized;
}
